tests/integration/hive_metastore/test_grants.py

```diff
@@ -12,7 +12,7 @@
 
 
 @retried(on=[NotFound, TimeoutError], timeout=timedelta(minutes=3))
-def test_all_grants_in_databases(runtime_ctx, sql_backend, make_group):
+def test_all_grants_in_databases(ws, runtime_ctx, sql_backend, make_group):
```

Comment:
Could you pick this up also?
```suggestion
def test_all_grants_in_databases(ws, runtime_ctx, sql_backend, make_group) -> None:
```

Reply:
Oops, I've backed this out: it was left over after reverting some changes made earlier in the PR.

Reply:
I think you still left some changes?